### PR TITLE
feat(movies): adds functionality for adding movies to database

### PR DESCRIPTION
**Pull request by kylekwong**
What: Lets users add movies to the database by sending a POST HTTP request.

Why: We want users to be able to add new movies so we can keep track of movies that are not already in our database.

Details:
- `movies` plugin, validator, and controller added.
- Tests for the `movies` validator and controller added in the `tests` folder.

### CODE REVIEW

File: test/plugins/features/movies/controller.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie = require('../../../../lib/models/movie');
```
Comment (elnaz) [inline test/plugins/features/movies/controller.test.js]: nit: align equals

**Comment by elnaz** [comment]
LGTM

**Comment by mgartner** [comment]
LGTM

File: test/plugins/features/movies/controller.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie      = require('../../../../lib/models/movie');
+
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
```
Comment (mgartner) [inline test/plugins/features/movies/controller.test.js:10]: Might be nice in this test to also check that it saves `release_year`.

**Pull request merged**